src/openzaak/components/documenten/api/serializers.py

```diff
@@ -396,51 +378,6 @@ def save(self, **kwargs):
         return self.instance
 
 
-class ObjectInformatieObjectSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
I think it was a mistake to delete the ObjectInformatieObject model, serializer and sync-methods.

Shouldn't they be still in place? We now removed a whole resource from the API, which is not allowed. I think the model etc should still be there. The sync methods can work on FK-level or whatever.

This keeps the API as-is and also keep functionality that will return anyway...

@sergei-maertens ?